### PR TITLE
lol

### PR DESCRIPTION
**Pull request by Rossmairm**

### CODE REVIEW

**Comment by skeswa** [comment]
lol

**Comment by Rossmairm** [comment]
That was for Dan! Now Merge my shiz!

File: lol.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+int  main(){
+printf("gophring things");
+
```
Comment (skeswa) [inline lol.c:9]: Such format.

File: lol.c [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
```
Comment (skeswa) [inline lol.c:2]: Very IO.

File: lol.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/python
+
+
+print("golfring things")
+
+
```
Comment (skeswa) [inline lol.py:9]: Much newline.

File: lol.py [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (skeswa) [inline lol.py:10]: wow.

**Pull request closed without merging**